src/pages/listing.tsx

```diff
@@ -8,17 +8,36 @@ import Head from 'next/head';
 const participationTimeComparator = (a: Participant, b: Participant) =>
   new Date(b.participationTime).getTime() - new Date(a.participationTime).getTime();
 
+const groupParticipantsByTheirLuck = (
```

Comment:
Minor note. The name of the function gives me the impression that there is a 'Luck' attribute, in the form of multiplier that affects the result of the 'draw' functionality, where in reality all of the participants have the same chance of winning. If I am not mistaken, the aim of the function is to separate the participants to groups based on whether a participant has won or not.

How about renaming it to something like `groupParticipantsBasedOnResult`. It indicates the separation of the participants, to two groups  based on the 'draw result' and is clearer, at least to my mind. Thoughts?

Reply:
Hello. Thanks for the comment and the suggestion.
Personally, I try to avoid adding a property's name to such functions' names. The reasons are that a property's name could either be paraphrased (i.e. data coming from a public API) or be not suitable to use for naming other things (i.e. _isWinner_). So, I try to see what a property expresses rather than reuse the name.

Also, I tried to use the term 'Their' in order to eliminate the confusion that 'Luck' indicates a property/attribute of the data. It seems I didn't achieve it :P 

Your suggestion looks fine to me. Adding also the term 'Draw' in the name we achieve 1. a clear name and 2. more friends from the java world.

_Goddess Luck is a multiplier that affects the result of a draw._ 🍀

Reply:
Thank you for taking the time to address my comment. 
The new naming, with the extra grain of salt you added (term 'Draw'), sounds even better to me 😁 

I am resolving this conversation 🤝